packages/botonic-react/src/components/index.d.ts

```diff
@@ -25,7 +27,9 @@ export interface MessageProps {
 
 export const Audio: React.FunctionComponent<MessageProps>
 export const Document: React.FunctionComponent<MessageProps>
+
 export const Message: React.FunctionComponent<MessageProps>
+export type Message = React.FunctionComponent<MessageProps>
```

Comment:
What's the difference between this line and the previous one and why exporting a const without assignment?

Reply:
this is something typical of typescript.
Message is both:
*  a thing (imagine const Message = (props => ...)) that you can pass as parameter of a function so that the function can create an jsx element from it 
* and a type (image that you have a list of buttons you would declare them as buttons: Button[])

Reply:
about the assignment, in d.ts you can declare types, but you can never assign values, as this is done in the .js files